koku/reporting_common/models.py

```diff
@@ -39,7 +47,9 @@ class Meta:
     provider = models.ForeignKey("api.Provider", on_delete=models.CASCADE)
     export_time = models.DateTimeField(null=True)
```

Comment:
In the design doc, we replace this with `export_datetime` for consistency, should that field be added in this PR as well?